loader/generated/vk_layer_dispatch_table.h

```diff
@@ -893,9 +900,6 @@ typedef struct VkLayerDispatchTable_ {
     // ---- VK_NV_fragment_shading_rate_enums extension commands
     PFN_vkCmdSetFragmentShadingRateEnumNV CmdSetFragmentShadingRateEnumNV;
 
-    // ---- VK_EXT_image_compression_control extension commands
-    PFN_vkGetImageSubresourceLayout2EXT GetImageSubresourceLayout2EXT;
-
```

Comment:
This is a little suspicious, but it looks like it just moved to the VK_EXT_host_image_copy section